src/emailpro/translations/Messages_fr_FR.xml

```diff
@@ -259,7 +259,7 @@
    <translation id="emailpro_ACTION_update_account_step1_display_name_label" qtlid="23515">Nom à afficher</translation>
    <translation id="emailpro_ACTION_update_account_step1_first_name_label" qtlid="23323">Prénom</translation>
    <translation id="emailpro_ACTION_update_account_step1_last_name_label" qtlid="3818">Nom</translation>
-   <translation id="emailpro_ACTION_update_account_step1_password_confirmation_label" qtlid="9983">Confirmation</translation>
+   <translation id="emailpro_ACTION_update_account_step1_password_confirmation_label">Confirmation</translation>
```

Comment:
Hello @varun257,

I'm not sure to understand what is the purpose of this change (and the only one changes in this PR) : 
If we remove a `qtlid` but have no change on the translation, the same `qtlid` will apply when we submit/retrieve translations, and this change will have no effect.

Reply:
Hi @cbourgois ,

The translation for this is not consistent across all files. In `en_US` it is `"Confirmation"` and in `en_CA/en_GB` it is `"Order Confirmation".`
The translation should be `"Confirmation"` in all english translations. Can you suggest how to make this consistent? Thanks

Reply:
And this issue seems to be breaking the UI as per the issue mentioned in Applicable issues.